app/views/users/registrations/new.html.erb

```diff
@@ -1,14 +1,7 @@
 <!--Descriptive page name, messages and instructions-->
 <h2 class="heading"><%= ts("Create Account") %></h2>
 <% if resource.errors.any? %>
```

Comment:
You _might_ be able to get rid of the surrounding `resource.errors.any?` logic as well -- could you check if you haven't?

Reply:
When I tried before it broke so I just put it back — I can investigate a little more into it to see if there's anything I can do about it though

Reply:
Hm I'm not sure what broke before but it works now haha — I'll get these changes pushed shortly